src/projects/tribute-page/index.html

```diff
@@ -1,20 +1,20 @@
-<div id="main">
+<main id="main">
   <h1 id="title">Dr. Norman Borlaug</h1>
-  <div>The man who saved a billion lives</div>
-  <div id="img-div">
+  <p>The man who saved a billion lives</p>
+  <figure id="figure">
     <img
       id="image"
       src="https://c2.staticflickr.com/4/3689/10613180113_fdf7bcd316_b.jpg"
-      alt="pic from flickr"
+      alt="Dr. Norman Borlaug seen standing in Mexican wheat field with a group of biologists"
     />
-    <div id="img-caption">
+    <figcaption id="figcaption">
```

Comment:
@lasjorg here should be `id="img-caption"`

Reply:
done